dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+
+        TestPluginPlugin.mojo().getLog().info("Extraneous Directories: " );
+        TestPluginPlugin.mojo().getLog().info("dtTestSource: " + dtTestSource);
+
+        //export DT_ROOT=/home/jgxue2/Test-Second-Bash-Script-V2
+        //export DT_SUBJ_ROOT=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d
+            // Top-most Module
+
+        //export DT_SUBJ_SRC=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core
+            // Desired Target Module
+        //export DT_SUBJ=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target
+            // Desired Module's Target Folder
+        //export DT_LIBS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/dependency/*
+            // Desired Target Module
+        //export DT_CLASS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/classes
+        //export DT_TESTS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/test-classes
+        //export DT_TEST_SRC=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/src/test/java
+    }
+
+    // Compile & Gather The Dependencies Of The Subject
+    private void gatherDependencies(){
+        TestPluginPlugin.mojo().getLog().info("Gathering Dependencies Of The Subject");
+
+        cli.doMain(new String[]{"compile",
+                "test-compile",
+                "-Dmaven.javadoc.skip=true",
+                "-DskipTests",
+                "dependency:copy-dependencies",
+                "-Drat.skip=true",
+                "-Dcobertura.skip" }, dtSubjectSource, System.out, System.out);
+        cli.doMain(new String[]{"install",
+                "-fn",
+                "-Dmaven.javadoc.skip=true",
+                "-DskipTests",
+                "dependency:copy-dependencies",
+                "-Drat.skip=true",
+                "-Dcobertura.skip" }, dtSubjectSource, System.out, System.out);
+    }
+
+    // Setup A Subject For Test Prioritization
+    private void setupTestPrioritization(MavenProject project){
+        TestPluginPlugin.mojo().getLog().info("Finding Human Written Tests (old version, original tests)");
+        try {
+            // Generates orig-order.txt
+            final List<String> tests = JavaConverters.bufferAsJavaList(TestLocator.tests(project).toBuffer());
+            FileWriter writer = new FileWriter(new File(dtResults + "/orig-order.txt"));
+            for(String str: tests) {
+                writer.write(str);
+                writer.write(System.getProperty("line.separator"));
+            }
+            writer.close();
+        } catch(Exception e){
+            e.printStackTrace();
+        }
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Locating Failed Tests");
+        args = new String[]{
```

Comment:
Do these args get used? They don't seem to (and if they are, they should be moved closer to the relevant location).